specs/waku/v2/waku-relay.md

```diff
@@ -22,11 +30,34 @@ authors: Oskar Thorén <oskar@status.im>
 
 **Protocol identifier***: `/vac/waku/relay/2.0.0-beta2`
 
+## Security Requirements
+
+<!-- In this part, we analyze the security of the  `relay` protocol concerning data confidentiality, integrity, authenticity, and anonymity. This is to enable users of this protocol to make an informed decision about all the security properties that can or can not achieve by deploying a `relay` protocol.-->
+
+- **Message Publisher Anonymity**: This property indicates that no adversarial entity can link a published `Message` to its origin i.e., the peer. Note that this feature also implies the unlinkability of the publisher to its published topic ID, this is because  `Message` contains the topic IDs as well.
+
+- **Topic Subscriber Anonymity:** This feature stands for the inability of any adversarial entity from linking a peer to its subscribed topic IDs.
+
+- **Confidentiality**: The adversary should not be able to learn the data carried by the `relay` protocol.
+-  **Integrity**: This feature indicates that the data transferred by the `relay` protocol can not be tampered with by an adversarial entity without being detected.
+-  **Authenticity**: No adversary can forge data on behalf of a targeted peer and make it accepted by other peers as if the origin is the target.
```

Comment:
Is this accurate, consider we do message signing?

Reply:
What does concern you about accuracy? If we sign messages then we preserve authenticity (should be careful about replay attack though)